pygeoapi/provider/wmts_facade.py

```diff
@@ -0,0 +1,278 @@
+                                          keywords, **kwargs)
+        else:
+            raise NotImplementedError(f"_{metadata_format.upper()}_ is not supported") # noqa
+
+    def get_html_metadata(self, dataset, server_url, layer, tileset,
+                          title, description, keywords, **kwargs):
+
+        service_url = url_join(
+            server_url,
+            f'collections/{dataset}/tiles/{tileset}/{{tileMatrix}}/{{tileRow}}/{{tileCol}}?f=png')  # noqa
+        metadata_url = url_join(
+            server_url,
+            f'collections/{dataset}/tiles/{tileset}/metadata')
+
+        metadata = dict()
+        metadata['id'] = dataset
+        metadata['title'] = title
+        metadata['tileset'] = tileset
+        metadata['collections_path'] = service_url
+        metadata['json_url'] = f'{metadata_url}?f=json'
+
+        return metadata
+
+    def get_default_metadata(self, dataset, server_url, layer, tileset,
+                             title, description, keywords, **kwargs):
+
+        service_url = url_join(
+            server_url,
+            f'collections/{dataset}/tiles/{tileset}/{{tileMatrix}}/{{tileRow}}/{{tileCol}}?f=png')  # noqa
+
+        content = {}
+        tiling_schemes = self.get_tiling_schemes()
+        # Default values
+        tileMatrixSetURI = tiling_schemes[0].tileMatrixSetURI
+        crs = tiling_schemes[0].crs
+        # Checking the selected matrix in configured tiling_schemes
+        for schema in tiling_schemes:
+            if (schema.tileMatrixSet == tileset):
+                crs = schema.crs
+                tileMatrixSetURI = schema.tileMatrixSetURI
+
+                tiling_scheme_url = url_join(
+                    server_url, f'/TileMatrixSets/{schema.tileMatrixSet}')
+                tiling_scheme_url_type = "application/json"
+                tiling_scheme_url_title = f'{schema.tileMatrixSet} tile matrix set definition' # noqa
+
+                tiling_scheme = LinkType(href=tiling_scheme_url,
+                                         rel="http://www.opengis.net/def/rel/ogc/1.0/tiling-scheme", # noqa
+                                         type=tiling_scheme_url_type,
+                                         title=tiling_scheme_url_title)
+
+        if tiling_scheme is None:
+            msg = f'Could not identify a valid tiling schema'  # noqa
+            LOGGER.error(msg)
+            raise ProviderConnectionError(msg)
+
+        content = TileSetMetadata(title=title, description=description,
+                                  keywords=keywords, crs=crs,
+                                  tileMatrixSetURI=tileMatrixSetURI)
+
+        links = []
+        service_url_link_type = "application/vnd.mapbox-vector-tile"
+        service_url_link_title = f'{tileset} raster tiles for {dataset}'
+        service_url_link = LinkType(href=service_url, rel="item",
+                                    type=service_url_link_type,
```

Comment:
should I fix it in the other providers as well?

Reply:
I noticed that this will cause the output of http://localhost:5000/collections/mapproxy/tiles/WebMercatorQuad/metadata?f=json to be like this:

```json
{
  "accessConstraints": "unclassified",
  "crs": "http://www.opengis.net/def/crs/EPSG/0/3857",
  "dataType": "vector",
  "description": "mapproxy",
  "links": [
    {
      "href": "http://localhost:5000/TileMatrixSets/WebMercatorQuad",
      "rel": "http://www.opengis.net/def/rel/ogc/1.0/tiling-scheme",
      "title": "WebMercatorQuad tile matrix set definition",
      "type_": "application/json"
    },
    {
      "href": "http://localhost:5000/collections/mapproxy/tiles/WebMercatorQuad/{tileMatrix}/{tileRow}/{tileCol}?f=png",
      "rel": "item",
      "title": "WebMercatorQuad raster tiles for mapproxy",
      "type_": "image/png"
    }
  ],
  "tileMatrixSetURI": "http://www.opengis.net/def/tilematrixset/OGC/1.0/WebMercatorQuad",
  "title": "mapproxy"
}
```

I don't know if that is desired. Maybe there needs to be a mapping in the json serialization? Or rename the property to in LinkType to `type`. I might be mistaken, though, and this is the desired output.